media-gfx/flameshot/flameshot-0.6.0.ebuild

```diff
@@ -22,6 +22,7 @@ DEPEND="
 	dev-qt/qtsvg:5
 	dev-qt/qtnetwork:5
 	dev-qt/qtdbus:5
+	dev-qt/linguist-tools:5
```

Comment:
With EAPI-7 this is a `BDEPEND`. Can you update your commit summary to match our requirements?
https://www.gentoo.org/glep/glep-0066.html#commit-messages

Reply:
It seems, that commit summary already matches the requirements

Reply:
> It seems, that commit summary already matches the requirements

not at all. just read the link provided to you.

Reply:
Oh, I see. Thanks, I didn't know about `repoman commit`.